app/models/company.rb

```diff
@@ -1,6 +1,25 @@
 class Company < ActiveRecord::Base
+  include ActiveModel::Dirty
```

Comment:
this module gives me access to the _changed? methods. Basically, I only wanted to geocode again if the city or state changes. This way we don't bust through the google api queries.